caluma/core/management/commands/cleanup_history.py

```diff
@@ -0,0 +1,42 @@
+from dateparser import parse
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from simple_history.models import registered_models
+
+
+class Command(BaseCommand):
+    """Cleanup historical records."""
+
+    help = "Cleanup historical records."
+
+    def add_arguments(self, parser):
+        parser.add_argument("--dry", dest="dry", action="store_true")
+        parser.add_argument(
+            "-k",
+            "--keep",
+            dest="keep",
+            default="",
```

Comment:
A little to the party, but shouldn't the `keep` option be required without a default?
As of now, if the argument has been forgotten, the whole history would be deleted, as the filter in L33 isn't applied.
Or am I mistaken?

Reply:
Good catch! I'll create an issue from it